website/README.md

```diff
@@ -1,12 +1,154 @@
-# The source of the Lima website (https://lima-vm.io)
+# Docsy Example
```

Comment:
Docsy shouldn’t be pushed to this repo, and it should be pulled from the upstream in netlify.toll

Reply:
The heading is wrong. Need to update readme for website code. Will do that 

Docsy is fetched as a module only. Its not pushed here